core/state/trie_prefetcher.go

```diff
@@ -77,15 +112,86 @@ func newTriePrefetcher(db Database, root common.Hash, namespace string) *triePre
 		storageWasteMeter: metrics.GetOrRegisterMeter(prefix+"/storage/waste", nil),
 	}
 	go p.abortLoop()
+	go p.mainLoop()
 	return p
 }
 
+func (p *triePrefetcher) mainLoop() {
+	for {
+		select {
+		case <-p.copyChan:
+			copy := &triePrefetcher{
```

Comment:
avoid 'copy'

Reply:
accepted, rename `copy` to `fetcherCopied`